idem_azurerm/exec/azurerm/resource/policy.py

```diff
@@ -138,7 +138,9 @@ async def assignment_create(hub, ctx, name, scope, definition_name, **kwargs):
                 scope=scope, policy_assignment_name=name, parameters=policy_model
             )
             result = policy.as_dict()
-        except CloudError as exc:
+        except (CloudError, ErrorResponseException) as exc:
```

Comment:
One of these things is not like the other. Any reason for the special handling here?

Reply:
not sure why the highlighting is different, but that's the same line.

Reply:
oh wait... you're talking about the lines below. I needed more information from a weird failure that would only happen on create.